.github/scripts/javascript/process-input.cjs

```diff
@@ -86,8 +89,20 @@ module.exports = async (context, github, core, inputs) => {
     const { issueId, command, issue } = await getIssueInfo(github, context, inputs);
     
     const isPullRequest = !!issue.data.pull_request;
-    const mode = (isPullRequest || command.startsWith('implement')) ? 'implementer' : 'refiner';
-    console.log(`Is PR: ${isPullRequest}, Mode: ${mode}`);
+    
+    // Determine mode based on explicit command first, then context
+    let mode;
+    if (command.startsWith('release-notes') || command.startsWith('release notes')) {
+      mode = 'release-notes';
+    } else if (command.startsWith('implement')) {
+      mode = 'implementer';
+    } else if (command.startsWith('refine')) {
+      mode = 'refiner';
+    } else {
+      // Default behavior when no explicit command: PR -> implementer, Issue -> refiner
+      mode = isPullRequest ? 'implementer' : 'refiner';
```

Comment:
non-blocker: the order of precedence is changing. before this pr, even if the command started with something as long as it was a `isPullRequest` then it would use the `implementer` mode. In this PR the order of precedence is given to the `command.startsWith`. I think it's actually better this way but calling it out.

Reply:
Great callout; I'm now interested to know how it behaves in a PR if you trigger it using one of the other ones.  

> I think it's actually better this way but calling it out.

 In theory I **do** like the explicit command taking priority, but not I'm not sure in practice it makes sense.   After merge I'll play around and see how I like it